features/step_definitions/planning_application_steps.rb

```diff
@@ -133,3 +133,12 @@
     expect(page).to have_selector("li", text: [question, response].join("\n"))
   end
 end
+
+Given("a created at audit entry") do
```

Comment:
This steps feels really artificial, is it used anywhere?

Reply:
will just take it out for now